.github/workflows/deploy.yml

```
@@ -8,7 +8,7 @@ jobs:
     name: New tag
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@trunk
```

```suggestion
      - uses: actions/checkout@v2
        with:
          ref: trunk
```

...maybe?

https://github.com/actions/checkout/tree/25a956c84d5dd820d28caab9f86b8d183aeeff3d#checkout-a-different-branch

---

Actually, I think it should just use the ref from the tag automatically if we use `actions/checkout@v2`

```suggestion
      - uses: actions/checkout@v2
```